crystals/tests/data/issue5_MgSiO3.direct.POSCAR

```diff
@@ -0,0 +1,28 @@
+Mg O3 Si
+1.0
+        4.4449000359         0.0000000000         0.0000000000
+        0.0000000000         4.6648001671         0.0000000000
+        0.0000000000         0.0000000000         6.4544000626
+    O   Mg   Si
+   12    4    4
+Direct
+     0.106900002         0.475499998         0.250000000
+     0.893100038         0.524500002         0.750000037
+     0.606899962         0.024500012         0.750000037
+     0.393099984         0.975499947         0.250000000
+     0.188300001         0.193900005         0.562099979
+     0.811700013         0.806099995         0.437900021
+     0.688299987         0.306100020         0.437900021
+     0.311699986         0.693900005         0.562099979
+     0.811700013         0.806099995         0.062099933
+     0.188300001         0.193900005         0.937899984
+     0.311699986         0.693900005         0.937899984
+     0.688299987         0.306100020         0.062099933
+     0.518199980         0.581700015         0.250000000
+     0.481800020         0.418299985         0.750000037
+     0.018199920         0.918299934         0.750000037
+     0.981799966         0.081700085         0.250000000
+     0.500000000         0.000000000         0.500000000
+     0.000000000         0.500000000         0.500000000
+     0.500000000         0.000000000         0.000000000
+     0.000000000         0.500000000         0.000000000
```

Comment:
Can you add a comment to say where this file comes from? If it has been converted from another file, can you write that down?

Reply:
These are converted from `issue5_MgSiO3.cif` with VESTA. I add this to the description at the first line of the file.